### ISSUE TITLE
[HOLD for payment 2024-08-19] [$250] Cannot search for emojis in status page

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#45629]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.8-3
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @Beamanator 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1721227885988609

## Action Performed:
1. Go to settings -> Status
2. Try to add search for an emoji to add for your status
## Expected Result:
Should be able to type in the Search and have emojis filter
## Actual Result:
Search field disabled and cannot type in Search
## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/6f690cfd-599d-4f2a-a639-7e07e501b1eb

https://github.com/user-attachments/assets/67e26196-df80-40be-b1bd-ff322d792b8f

Add any screenshot/video evidence

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01298542065fc43b99</li>
        <li>Upwork Job ID: 1816515758396555142</li>
        <li>Last Price Increase: 2024-07-25</li>
<li>Automatic offers: </li>
<ul>
<li>situchan | Reviewer | 103345487</li>
<li>Krishna2323 | Contributor | 103345488</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @stephanieelliott</details>

**Comment by FitseTLT** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Cannot search for emojis in status page
### What is the root cause of that problem?
Because we are displaying the popover without overlay here
https://github.com/Expensify/App/blob/7ba66a1542fa815eed5cb4622244bc2935593484/src/components/EmojiPicker/EmojiPicker.tsx#L213
### What changes do you think we should make in order to solve the problem?

We shouldn't pass `withoutOverlay` as true, so we should remove this line
https://github.com/Expensify/App/blob/7ba66a1542fa815eed5cb4622244bc2935593484/src/components/EmojiPicker/EmojiPicker.tsx#L213

### What alternative solutions did you explore? (Optional)

**Comment by FitseTLT** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Cannot search for emojis in status page
### What is the root cause of that problem?
Because we are displaying the popover without overlay here
https://github.com/Expensify/App/blob/7ba66a1542fa815eed5cb4622244bc2935593484/src/components/EmojiPicker/EmojiPicker.tsx#L213
### What changes do you think we should make in order to solve the problem?

We shouldn't pass `withoutOverlay` as true, so we should remove this line
https://github.com/Expensify/App/blob/7ba66a1542fa815eed5cb4622244bc2935593484/src/components/EmojiPicker/EmojiPicker.tsx#L213

Result:

https://github.com/user-attachments/assets/4fc13bd1-6ef9-4f21-8e4a-0173eec6041d


### What alternative solutions did you explore? (Optional)

**Comment by stephanieelliott** [comment]
Gonna close this as `Not a priority` as it does not fit into our current roadmap. May reopen in the future.

**Comment by mallenexpensify** [comment]
I think we should fix, just happened to me.  Searching for emoji via text for status updates is common (3 folks reported bug)

**Comment by situchan** [comment]
Thanks for the proposals. Can you please add in RCA the reason why this doesn't happen in composer / add reaction emoji picker but only in status page?

**Comment by stephanieelliott** [comment]
PR is on staging 🎉

**Comment by stephanieelliott** [comment]
Hey @situchan can you please complete the BZ checklist when you get a chance?

**Comment by situchan** [comment]
> **BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:
> 
> * [x]  [@situchan] The PR that introduced the bug has been identified. Link to the PR: https://github.com/Expensify/App/pull/39520
> * [x]  [@situchan] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment: https://github.com/Expensify/App/pull/39520#pullrequestreview-2244640401
> * [x]  [@situchan] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion: N/A
> * [x]  [@situchan] Determine if we should create a regression test for this bug.
> * [x]  [@situchan] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.

### Regression Test Steps
1. Go to settings -> Status > Open emoji picker
2. Verify that input is focused and can be used for searching emojis
3. Verify tab/up/down keys work as expected

**Comment by stephanieelliott** [comment]
Summarizing payment on this issue: 

 - Contributor: @Krishna2323 $250 via Upwork - PAID 
 - Contributor+: @situchan $250 via Upwork - PAID 
 
 Upwork job is here: https://www.upwork.com/jobs/~01298542065fc43b99

**Comment by stephanieelliott** [comment]
Issue for test rail case here: https://github.com/Expensify/Expensify/issues/421648
